src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -38,6 +41,8 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
   const { user } = useUser();
 
   const { selectRecipientChat } = useChatSidebarControls();
+  const { recentWorldUsers } = useRecentWorldUsers();
```

Comment:
@sparkletown/product @sparkletown/design @mike-lvov Also, instead of just `online` / `offline`, should we be showing any/all of their status types here?

Reply:
https://github.com/sparkletown/sparkle/pull/1484/commits/c91da9a9f91b044f1211459a072d3dc813ecdf54